Automattic/wp-calypso
# Move Email Forwarding to Email Module

#### Changes proposed in this Pull Request

* Move the email forwarding components to the email module
* Remove relevant css from the domains management style file into a new email forwarding style file

#### Testing instructions

1. Navigate to email forwarding on a domain without g suite
1. Verify that you can
   1. Add an email forward
   1. Remove an email forward
   1. Resend the verification email for an unverified email forward
